### ISSUE TITLE
Trying PoinTr on a custom dataset

### ISSUE THREAD

**Issue by Mona-ShZeinoddin** [yuxumin/PoinTr#64]
Hello and congrats on your work!

I have my own dataset which I would like to train PoinTr on, the dataset consists of one category of objects and I am following the implementation of PCN to configure my dataset. The result I get is quite surprising, first of all, although I only have one category, I only have a F-score of 4 percent, Second, I have the same pattern in all of my test results, a group of outliers in the prediction. Do you have any suggestions as to how to improve the output?(hyper parameter tuning, the way I should normalize data(I have normalized the same as PCN [-0.5,0.5]
![Screenshot from 2022-06-27 15-24-44](https://user-images.githubusercontent.com/101371071/176230657-f6cc593d-1ecd-426f-8c72-c895b694d265.png)
, etc.)

**Comment by Mona-ShZeinoddin** [comment]
Yes it is trained from scratch for 1000 epochs on a dataset containing 1000 samples(the data contains simple branches and doesn't contain details such as other datasets)

**Comment by yuxumin** [comment]
I see. I guess the issue may come from underfitting of the model due to limited training data. Could you try to initialize the model with the weights of PoinTr pretrained on PCN dataset? 
BTW, the two lines in your figure is the partial point cloud for a sample?

**Comment by Mona-ShZeinoddin** [comment]
Thank you I will try it out right away,

No, the blue line is the ground truth and the green is the predicted, as you can see there are a number of outliers in the prediction and also the prediction and the ground truth don't have a high intersection area.

**Comment by Mona-ShZeinoddin** [comment]
BTW, can you justify why the F-score is 4 percent and not one, even though I only have one class?

**Comment by 8umpk1n** [comment]
Hello, I encountered a lot of problems when I made my own dataset for training, can I ask you for advice?

**Comment by yuxumin** [comment]
@8umpk1n 
see https://github.com/yuxumin/PoinTr/issues/34
